Change: valueX => valueY

@package vaimo/composer-patches-target1
@after second-change

--- src/example.txt.org
+++ src/example.txt
@@ -5,7 +5,7 @@
 value5
 value6
 value7
-valueX
+valueY
 value9
 value10
 value11
